packages/cubejs-schema-compiler/src/adapter/BaseQuery.js

```diff
@@ -2845,6 +2845,9 @@ export class BaseQuery {
       return evaluateSql === '*' ? '1' : evaluateSql;
     }
     if (this.ungrouped) {
+      if ((this.safeEvaluateSymbolContext().ungroupedAliases || {})[measurePath]) {
```

Comment:
```suggestion
      if (this.safeEvaluateSymbolContext().ungroupedAliases?[measurePath]) {
```

Reply:
This is an invalid syntax, so leaving it as is.